packages/image_picker/image_picker/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -84,16 +84,28 @@ - (void)start {
   }
   if (@available(iOS 14, *)) {
     [self setExecuting:YES];
-
-    if ([self.result.itemProvider hasItemConformingToTypeIdentifier:UTTypeWebP.identifier]) {
-      [self.result.itemProvider
-          loadDataRepresentationForTypeIdentifier:UTTypeWebP.identifier
-                                completionHandler:^(NSData *_Nullable data,
-                                                    NSError *_Nullable error) {
-                                  UIImage *image = [[UIImage alloc] initWithData:data];
-                                  [self processImage:image];
-                                }];
-      return;
+    NSArray *supportedRepresentations = @[
```

Comment:
"Supported" seems misleading here; this isn't an exhaustive list since there's the other codepath as well. What is distinct about these vs other types that makes them need to use a different code path (so that we can name the list accordingly)?